### PR TITLE
clusterloader2: drop metrics deprecated in 1.19

### PR DESCRIPTION
**Pull request by ingvagabund**
Both predicate_evaluation and priority_evaluation operation metrics are replaced
with filter, resp. score extension points.
Also, preemption_evaluation, resp. binding operation metrics are replaced with
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket, resp.
scheduler_binding_duration_seconds_bucket metrics.

### CODE REVIEW

**Comment by ingvagabund** [comment]
Collected metrics contain the following entries for binding and predicate evaluation:
```
# HELP scheduler_binding_duration_seconds [ALPHA] Binding latency in seconds
# TYPE scheduler_binding_duration_seconds histogram
scheduler_binding_duration_seconds_bucket{le=\"0.001\"} 0
scheduler_binding_duration_seconds_bucket{le=\"0.002\"} 0
scheduler_binding_duration_seconds_bucket{le=\"0.004\"} 41
scheduler_binding_duration_seconds_bucket{le=\"0.008\"} 633
scheduler_binding_duration_seconds_bucket{le=\"0.016\"} 1707
scheduler_binding_duration_seconds_bucket{le=\"0.032\"} 2852
scheduler_binding_duration_seconds_bucket{le=\"0.064\"} 3806
scheduler_binding_duration_seconds_bucket{le=\"0.128\"} 4805
scheduler_binding_duration_seconds_bucket{le=\"0.256\"} 6157
scheduler_binding_duration_seconds_bucket{le=\"0.512\"} 7037
scheduler_binding_duration_seconds_bucket{le=\"1.024\"} 7282
scheduler_binding_duration_seconds_bucket{le=\"2.048\"} 7282
scheduler_binding_duration_seconds_bucket{le=\"4.096\"} 7282
scheduler_binding_duration_seconds_bucket{le=\"8.192\"} 7282
scheduler_binding_duration_seconds_bucket{le=\"16.384\"} 7282
scheduler_binding_duration_seconds_bucket{le=\"+Inf\"} 7282
scheduler_binding_duration_seconds_sum 887.1218679819982
scheduler_binding_duration_seconds_count 7282
# HELP scheduler_scheduling_algorithm_preemption_evaluation_seconds [ALPHA] Scheduling algorithm preemption evaluation duration in seconds
# TYPE scheduler_scheduling_algorithm_preemption_evaluation_seconds histogram
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.001\"} 0
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.002\"} 0
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.004\"} 26
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.008\"} 38
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.016\"} 50
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.032\"} 55
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.064\"} 60
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.128\"} 63
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.256\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"0.512\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"1.024\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"2.048\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"4.096\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"8.192\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"16.384\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_bucket{le=\"+Inf\"} 64
scheduler_scheduling_algorithm_preemption_evaluation_seconds_sum 1.060999083
scheduler_scheduling_algorithm_preemption_evaluation_seconds_count 64
```

**Comment by ingvagabund** [comment]
@ahg-g PTAL

**Comment by ingvagabund** [comment]
As per https://github.com/kubernetes/perf-tests/pull/1081#discussion_r403112191 comment, do you want to display the deprecated binding/preemption metrics with the new ones? Right now, the old metrics are just replaced by the new ones.

File: clusterloader2/pkg/measurement/common/scheduler_latency.go [hunk 1]
```diff
@@ -42,7 +41,8 @@ const (
 	e2eSchedulingDurationMetricName           = model.LabelValue(schedulermetric.SchedulerSubsystem + "_e2e_scheduling_duration_seconds_bucket")
 	schedulingAlgorithmDurationMetricName     = model.LabelValue(schedulermetric.SchedulerSubsystem + "_scheduling_algorithm_duration_seconds_bucket")
 	frameworkExtensionPointDurationMetricName = model.LabelValue(schedulermetric.SchedulerSubsystem + "_framework_extension_point_duration_seconds_bucket")
-	schedulingLatencyMetricName               = model.LabelValue(schedulermetric.SchedulerSubsystem + "_" + schedulermetric.DeprecatedSchedulingDurationName)
+	bindingDurationMetricName                 = model.LabelValue(schedulermetric.SchedulerSubsystem + "_binding_duration_seconds_bucket")
```
Comment (ahg-g) [inline clusterloader2/pkg/measurement/common/scheduler_latency.go:44]: binding is already one of the extension points, so we don't really need a separate metric for it.

**Comment by ingvagabund** [comment]
@wojtek-t PTAL

**Comment by ingvagabund** [comment]
>  https://go.k8s.io/oncall

@mm4tt can you please reload perfdash?

**Pull request merged**